ui/components/app/modals/nickname-popovers/nickname-popovers.component.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
```

Comment:
This file controls the display of the two nickname popovers:

![Screenshot from 2021-12-01 09-40-45](https://user-images.githubusercontent.com/7499938/144240300-e198ca51-926c-4e23-9641-33ff5711b890.png)
![Screenshot from 2021-12-01 09-40-41](https://user-images.githubusercontent.com/7499938/144240301-aa85ed3e-56d0-4d41-a07a-1537bbde03db.png)